cross/i686-w64-mingw32-gcc/Portfile

```diff
@@ -8,7 +8,7 @@ set mingw_name      w64-mingw32
 set mingw_arch      i686
 set mingw_target    ${mingw_arch}-${mingw_name}
 
-crossgcc.setup      ${mingw_target} 11.3.0
+crossgcc.setup      ${mingw_target} 12.1.0
```

Comment:
Going off gcc-devel port updated last month we should be fine down to at least darwin12.

It may just work below this also as mingw-gcc requires are different than regular gcc.